doc/CMakeLists.txt

```diff
@@ -16,7 +16,9 @@ configure_file(
 
 add_custom_target(
     doc ALL
+    COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/fetch-xeplist.sh
     COMMAND ${DOXYGEN_EXECUTABLE} ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile
+    COMMAND rm ${CMAKE_CURRENT_SOURCE_DIR}/xeplist.xml
     WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
     COMMENT "Generate API documentation with Doxygen"
     VERBATIM
```

Comment:
* have you tested this on windows?
* this way it's not possible to generate the documentation when you're offline, right? debian requires that building works offline (and I think that's a good idea)

Reply:
The XEP list is now a local copy in QXmpp's repository and can be updated by `doc/update-xeplist.sh`.